src/utils/user_utils.ts

```diff
@@ -171,3 +173,30 @@ export function sortByUsername(a: UserProfile, b: UserProfile): number {
 
     return nameA.localeCompare(nameB);
 }
+
+export function applyRolesFilters(user: UserProfile, filterRoles: string[], membership?: TeamMembership | ChannelMembership): boolean {
+    const roles = user.roles;
```

Comment:
Do we need this? Can we just user `user.roles` in place of `roles` as it only seems to be called 3 times so the extra variable doesn't seem to make too much sense.